build.yml

```
@@ -60,6 +60,30 @@ docker-targets:
       xfonts-base
       zlib1g
 
+  bookworm:
+    source: docker/Dockerfile.debian
+    args:
+      from: debian:bookworm
+      python: python3
+    output: deb
+    matrix: ['amd64', 'arm64']
```

Can you keep it the same as `bullseye` i.e
```suggestion
    matrix: ['amd64', 'i386', 'arm64', 'armhf', 'armel', 'ppc64el']
```
assuming that docker images are available for all the above architectures?